src/librustc_ast_passes/ast_validation.rs

```diff
@@ -775,13 +775,13 @@ fn validate_generic_param_order<'a>(
         err.span_suggestion(
             span,
             &format!(
-                "reorder the parameters: lifetimes, then types{}",
-                if sess.features_untracked().const_generics
-                    || sess.features_untracked().min_const_generics
-                {
-                    ", then consts"
+                "reorder the parameters: lifetimes{}",
+                if sess.features_untracked().const_generics {
+                    ", then consts and types"
+                } else if sess.features_untracked().min_const_generics {
+                    ", then consts, then types"
```

Comment:
Shouldn't this be `", then types, then consts"`?

Reply:
ups

Reply:
@JulianKnodt do you want to open a PR for this, otherwise I will quickly fix that?